btrdbextras/eventproc/protobuff/api.proto

```diff
@@ -17,7 +17,7 @@ message Handler {
     string name = 3;
     int32 version = 4;
     string callable = 5;
-    repeated string flag = 6;
+    repeated string tag = 6;
```

Comment:
If we havent recompiled the grpc files yet with `protoc` and the newer versions of grpc, we should do that before merging.